packages/components/src/ToggleSwitch/ToggleSwitchField/ToggleSwitchField.tsx

```diff
@@ -59,7 +59,6 @@ export const ToggleSwitchField = ({
         >
           <ToggleSwitch
             id={`${id}-field-toggle`}
-            disabled={disabled}
```

Comment:
This one will be required because it gets passed through to the `input`

It's the `disabled` prop on `<Label>` above that you'll want to get rid of